### PR TITLE
Scripts for downloading, installing and running a Solr 9 Cloud

### PR DESCRIPTION
**Pull request by tokee**
Scripts ported and upgraded from Solr 8 to Solr 9. README contains instructions for use.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,11 +47,29 @@ the README, using the Solr setup from `nested-template/conf/`. This boils down t
 ```shell
```
Comment (VictorHarbo) [inline README.md:47]: The branch is named `nested-solr-indexing'. Also I misread the readme and thought that I had to start solr from [ds-solr](https://github.com/kb-dk/ds-solr/). However, thats not the case. Maybe specify that a more thorough documentation can be found there, but these three lines should do the trick in ds-present?

**Review by VictorHarbo — COMMENTED** [review]
The scripts work. However I got a little confused by the documentation as it refers to ds-solr and checking out a branch from that project. I read it as i had to be located in ds-solr to call the scripts mentioned in the readme here. 

Scripts worked flawless when run from the correct place.

File: README.md (hunk 1 above)
Comment (tokee) [reply README.md:47]: Whoops, leftover from old instructions.

**Review by VictorHarbo — APPROVED** [review]
Looks good to me!

**Pull request merged**